Dev 2.11

fc7 opened the pull request:
Trivial fix to restore missing imports in the MacAdapter

simonharrer commented:
Thank you for your pull request. Can you please add yourself to the list of contributors according to the CONTRIBUTORS instructions?

fc7 commented:
Of course, will do!

fc7 commented:
I have added my name to the list of contributors in About.html
I have added some more minor commits to improve code readability (which also led to a minor bugfix).
